python/requirements.txt

```diff
@@ -1,2 +1,11 @@
-# This file should list any python package dependencies.
-nose==1.3.3
\ No newline at end of file
+# Test
+nose>=1.3
+
+# Lint
+pep8>=1.7
+flake8>=2.5
+pyflakes>=1.2
+
+# Packaging
```

Comment:
@thunterdb This packages are pretty stable by now… but in case of API changes I pinned theirs major versions.